redbeams-api/src/main/java/com/sequenceiq/redbeams/doc/ModelDescriptions.java

```diff
@@ -45,6 +45,42 @@
         public static final String DATABASE_SERVER = "The external database server configuration";
     }
 
+    public static class DBStack {
+        public static final String STACK_NAME = "Name of the database stack";
+        public static final String REGION = "Region of the database stack";
+        public static final String NETWORK = "Network for the database stack";
+        public static final String DATABASE_SERVER = "Database server for the database stack";
+        public static final String AWS_PARAMETERS = "AWS-specific parameters for the database stack";
+    }
+
+    public static class NetworkModelDescription {
+        public static final String AWS_PARAMETERS = "AWS-specific parameters of the specified network";
```

Comment:
NIT: inconsistent use of "for" and "of" throughout this file.

Reply:
I was trying to follow what other description strings do, but it is definitely not worth it. Changing over to "of" in most places.